code/game/objects/items/blueprints.dm

```diff
@@ -65,8 +65,8 @@
 
 
 /obj/item/areaeditor/permit/create_area()
-	..()
-	qdel(src)
+	if(..() == TRUE)
```

Comment:
Just FYI - this doesn't need to do ` == TRUE` - `if()` will automatically execute the bit in the code block if the thing it's evaluating isn't null. This means that if `..()` returns any value, the if proc will do its thing, so the `==TRUE` bit is unnecessary!

Reply:
Ah thank you for the tip! I'll be sure to remember in the future!